src/duals.jl

```diff
@@ -58,10 +58,10 @@ function _compute_parameters_in_ci!(
 end
 
 function _compute_parameters_in_ci!(
-    model::Optimizer{T},
-    pf,
+    model::Optimizer{T, OT},
+    pf::ParametricAffineFunction{T},
     ci::MOI.ConstraintIndex{F,S},
-) where {F,S} where {T}
+) where {F,S} where {T, OT}
```

Comment:
I think OT is not needed in any of these methods.
T is important to gave matching types.

Reply:
unfortunately OT is needed because the optimizer has two types inside.
```julia
ParametricOptInterface.Optimizer{Float64, GLPK.Optimizer}
```
T catches the Float64 and OT catches the solver.